unique_sdk/unique_sdk/_http_client.py

```diff
@@ -7,7 +7,7 @@
 except ImportError:
     requests = None
 
-from unique_sdk import _error
+from unique_sdk import _error, unique_sdk
```

Comment:
This looks wrong. I think you want a separate import. 
```suggestion
import unique_sdk
from unique_sdk import _error
```